### ISSUE TITLE
fxFlex not fill the parent in v6.0.0-beta.15

### ISSUE THREAD

**Issue by banminkyoz** [angular/flex-layout#749]
#### Bug, feature request, or proposal:

After i upgrade to angular 6, i have to update rxjs and flex-layout too. And with flex-layout 6.0.0-beta-15 it seem my UI are all broke :cry: 

I also know that there are breaking changes, i'v also tried some workaround but it all not work :sob:

With
```css
[fxFlex] {
  display: flex; 
}
```
it seem fix some problem, but sometime i use fxFill too, and fxFill is still facing similar problem. is there any way i can fix it ?

#### What is the expected behavior?

Back in 5.1.1. This is most used layout on our page
https://stackblitz.com/edit/ng-flex-layout-5

#### What is the current behavior?

And now it become this:
https://stackblitz.com/edit/ng-flex-layout-6
(maybe the height or min-height is the problem)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

os: ubuntu 18.04, OSX high sierra
browser: chrome
angular: 6.0.2
flex-layout: 6.0.0-beta.15
typescript: 2.7.2

**Comment by julianobrasil** [comment]
@banminkyoz, there was a breaking change over here: #670

**Comment by banminkyoz** [comment]
Thanks @julianobrasil , as read those reference issues i find out the way to make my app back to original state. :smile: 

```css
[fxFlex] {
  display: flex; 
  flex-direction: row;
}
*[fxlayout='column'] > *[fxFlex] {
  flex-basis: 0.000000001px !important;
}
```

**Comment by CaerusKaru** [comment]
This will be fixed by 6.0.0-beta.16 and is tracked by #728
